app/_landing_pages/insomnia/collections.yaml

```
@@ -0,0 +1,162 @@
+    columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: | 
+                    With Insomnia, you can send requests via HTTP, gRPC, GraphQL, and WebSockets. You can send a single request or run a series of requests in a specific order.
+          - type: button
+            config:
+              align: left
+              text: Learn more about requests
+              url: /insomnia/requests/
+      - blocks:
+        - type: image
+          config:
+            url: /assets/images/insomnia/request.png
+            alt_text: Image of a request
+  - header:
+      type: h3
+      text: "Get started with requests"
+    columns:
+      - blocks:
+        - type: card
+          config:
+            title: Use the Collection Runner
+            cta:
+              text: Learn more
+              url: /how-to/use-the-collection-runner/
+              align: end
+      - blocks:
+        - type: card
+          config:
+            title: Chain requests
+            cta:
+              text: Learn more
+              url: /how-to/chain-requests/
+              align: end
+ 
+  - header:
+      type: h2
+      text: "Write scripts"
+    columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: | 
+                    The pre-request and after-response scripts allows you to interact with the request and its response. You can use these to:
+                - type: unordered_list
+                  items:
+                  - Set or clear variables
+                  - Update the contents of the request
+                  - Send other requests
+                  - Run tests on the response
+          - type: button
+            config:
+              align: left
+              text: Learn more about scripts
+              url: /insomnia/scripts/
+      - blocks:
+        - type: image
+          config:
+            url: /assets/images/insomnia/after-response-script.png
+            alt_text: Image of an after-response script
+  - header:
+      type: h3
+      text: "Get started with scripts"
+    columns:
+      - blocks:
+        - type: card
+          config:
+            title: Write pre-request scripts to add dynamic behavior
+            cta:
+              text: Learn more
+              url: /how-to/write-pre-request-scripts/
+              align: end
+  - header:
+      text: "Frequently Asked Questions"
+      type: h2
+    columns:
+      - blocks:
+        - type: faqs
+          config:
+            - q: What's the difference between a collection and a design document?
+              a: Documents are a data object that can contain an API spec, a collection of API requests, and tests. Collections are simply a series of API requests that you intend to interact with Insomnia.
+            - q: Can I import requests from another tool?
```

```suggestion
            - q: Can I import requests from another tool?
```
Ill add a link to the import page in my PR when it is merged.